gnmidiff/set_to_get.go

```diff
@@ -15,18 +15,32 @@ import (
 // by ygot.PathToString.
 // - The value of the update fields is the JSON_IETF representation of the
 // value.
-// TODO: Format function
 type SetToNotifsDiff struct {
-	// MissingUpdates are updates specified in the SetRequest missing in
+	// MissingUpdates (-) are updates specified in the SetRequest missing in
 	// the input Notifications.
 	MissingUpdates map[string]interface{}
-	// ExtraUpdates are updates not specified in the SetRequest that's
+	// ExtraUpdates (+) are updates not specified in the SetRequest that's
 	// present in the input Notifications.
 	ExtraUpdates      map[string]interface{}
 	CommonUpdates     map[string]interface{}
 	MismatchedUpdates map[string]MismatchedUpdate
 }
 
+// Format outputs the SetToNotifsDiff in human-readable format.
+//
+// NOTE: Do not depend on the output of this being stable.
+func (diff SetToNotifsDiff) Format(f Format) string {
+	f.title = "SetToNotifsDiff"
+	f.aName = "want/SetRequest"
+	f.bName = "got/Notifications"
+	return SetRequestIntentDiff{
```

Comment:
i'd extract the common logic of Format into its own struct instead using the SetRequestIntentDiff struct

```
type StructuredDiff struct {} 
func (fd StructuredDiff) Format()

type SetToNotifsDiff struct { StructuredDiff } 
type SetRequestIntentDiff struct { StructuredDiff } 
```

Reply:
Made more modular, PTAL